pkgs/servers/nextcloud/packages/25.json

```diff
@@ -80,19 +90,19 @@
     ]
   },
   "mail": {
-    "sha256": "161ksx7g32fkpwxq3vij2vw6sxblv7xrxggsxnx2wj14ik3wxclv",
-    "url": "https://github.com/nextcloud-releases/mail/releases/download/v2.3.0-alpha.4/mail-v2.3.0-alpha.4.tar.gz",
-    "version": "2.3.0-alpha.4",
+    "sha256": "0sk19y9iwnbw6ci4hq9xxrn5qgnlgqlb5hpzc50i706s1z5n9cmc",
+    "url": "https://github.com/nextcloud-releases/mail/releases/download/v2.2.4/mail-v2.2.4.tar.gz",
+    "version": "2.2.4",
     "description": "**💌 A mail app for Nextcloud**\n\n- **🚀 Integration with other Nextcloud apps!** Currently Contacts, Calendar & Files – more to come.\n- **📥 Multiple mail accounts!** Personal and company account? No problem, and a nice unified inbox. Connect any IMAP account.\n- **🔒 Send & receive encrypted mails!** Using the great [Mailvelope](https://mailvelope.com) browser extension.\n- **🙈 We’re not reinventing the wheel!** Based on the great [Horde](https://horde.org) libraries.\n- **📬 Want to host your own mail server?** We do not have to reimplement this as you could set up [Mail-in-a-Box](https://mailinabox.email)!",
     "homepage": "https://github.com/nextcloud/mail#readme",
     "licenses": [
       "agpl"
     ]
   },
   "news": {
-    "sha256": "17kz5499jkv43w8wcd1p982hpkw2akgzpv9cjj8qqjhvzv4qr171",
-    "url": "https://github.com/nextcloud/news/releases/download/21.0.0-beta1/news.tar.gz",
-    "version": "21.0.0-beta1",
+    "sha256": "1j7iqnjm1f1kqhhfw1kc36a740mnlkwf4yn2rszlkdgizmfcwvra",
+    "url": "https://github.com/nextcloud/news/releases/download/21.0.0/news.tar.gz",
+    "version": "21.0.0",
```

Comment:
this is also a downgrade

Reply:
No, 21.0.0 comes after the beta: https://github.com/nextcloud/news/releases